common/Makefile.common.mk

```diff
@@ -25,6 +25,7 @@
 BUILD_LOCALLY ?= 1
 IMAGE_NAME ?= ibm-crossplane
 RELEASE_VERSION ?= $(shell cat RELEASE_VERSION)
+GO_SUPPORTED_VERSIONS = 1.14|1.15
```

Comment:
is this necessary? Golang is strict with backwards compatibility, so if go 1.14 works, then go 1.15 is implied to also work

Reply:
crossplane build script only supports up to 1.14 .. the variable is to override crossplane build script to use go 1.15

Reply:
the whole common directory is added by IBM